libraries/chain/code_object.cpp

```diff
@@ -0,0 +1,14 @@
+#include <eosio/chain/code_object.hpp>
+#include <eosio/chain/wasm_interface.hpp>
+
+namespace eosio { namespace chain {
+
+void code_object::reflector_init()
+{
+   static_assert(fc::raw::has_feature_reflector_init_on_unpacked_reflected_types,
+                 "code_object expects FC to support reflector_init");
```

Comment:
We really don't need these `static_assert`s anymore now that `libfc` is no longer a submodule. It is fine to leave it, but we should probably clean those up at some point.